uurshin/minotaure
# feat(translation)

## Description

> I'm quickly proposing this mini PR because I noticed a key error in the previous one.

* Update some translation keys


## Test

* `npm run build`
* `npm run preview`
* go to [home](http://localhost:4173/)

* navigate through the app

* check for missing translations
* read the code
